spec/src/main/asciidoc/ssl.asciidoc

```diff
@@ -0,0 +1,91 @@
+//
+// Copyright (c) 2019 Contributors to the Eclipse Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+[[ssl]]
+== MicroProfile Rest Client SSL Support
+
+MicroProfile Rest Client provides a uniform way to configure
+SSL for the client.
+
+=== Trust store
+By default, a MicroProfile Rest Client implementation uses the
+JVM trust store.
+MicroProfile Rest Client provides a way to specify a
+custom trust store.
+
+For clients created programmatically, the trust store should be read to
+a `KeyStore` object and specified as follows:
+
+[source, java]
+----
+KeyStore trustStore = readTrustStore();
+RestClientBuilder.newBuilder()
+    .trustStore(trustStore)
+
+----
+
+
+For CDI injected clients, the trust store can be specified
+with MicroProfile Config properties:
+
+- `myClient/mp-rest/trustStore` to set the trust store location.
+Can point to either a classpath resource (e.g. `classpath:/client-truststore.jks`)
+or a file (e.g. `file:/home/user/client-truststore.jks`)
+- `myClient/mp-rest/trustStorePassword` to set the
```

Comment:
Do other MP specs require a password in config? Wondering if there's anything we need to do around plain text passwords in config?

Reply:
I don't know of any...